docs/project/faq.md

```diff
@@ -310,6 +313,84 @@ actively encourage using these languages in those cases. However, we need a
 solution for C++ use-cases that require its full performance, low-level control,
 and access to hardware.
 
+## Why is Carbon designed that way?
+
+### Why are there no angle brackets?
```

Comment:
> Angle brackets exist for comparison operators

To me, "angle bracket" refers specifically to using those tokens as part of a balanced-pair syntax, but I've rephrased to avoid the confusion.

> it's probably going to be helpful to be specific about the feature.

I'm trying to explain why we don't use angle brackets _anywhere_, not just why we don't use them for templates and generics.

> I think it can be helpful to be more positive in the phrasing of questions.

I don't disagree, but in light of the above I don't see a positive way to phrase this question.